data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,370 @@
+	description "This ship is now appropriate as an agile, long distance, cargo hauler and blockade runner. It is able to defend itself in small skirmishes, even though you suspect that it does not operate at its full potential, as if it stayed underwater for some time... The simple fact that it is still running at all after so many years is clearly proof of the durability of Hai technology."
+
+mission "Hai Legacy 1"
+	invisible
+	landing
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		not "event: hai ladybug available"
+	on offer
+		conversation
+			`As you enter the shipyard you quickly get approached by someone wearing a hood, apparently not too keen to reveal their identity. They come to address you, discreetly. Seeing you stop to listen, they take your arm and make you follow them. "Keep walking," they say, quickly taking a look around the way someone looking for a new ship would, before addressing you again. "My associates and I have a business proposition for you, if you are interested to learn more about it go to these coordinates." They give you a piece of paper containing the location, on a beach almost on the other side of the planet with an instruction to not go with you ship, before quickly fade in the crowd. Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			`	You proceed to hire a motorized antigrav vehicle to go there, and by the time you arrive it starts being dark. You set foot after double checking you are indeed at the right coordinates, for except a cave, vegetation, sand, and, of course, the beach, you cannot see much of interest in the vicinity. You can't help but put your hand close to your gun, feeling safer with its presence.`
+			`	After waiting for what seems like an eternity you start to hear whisper, and two hooded figures approach you. You start to doubt whether or not this was a good idea but it is probably too late to back down now. They come to address you, their face hidden by the now almost total obscurity. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious mood. "We are sorry for all of these complications, but the truth is we cannot afford to take risks. It has been some time since we stopped trying to understand, and started taking every precaution we could to be safe. You see, we simply do not know you, and neither do you us."`
```

Comment:
```suggestion
			`	After waiting for what seems like an eternity you start to hear whispering, and two hooded figures approach you. You start to doubt whether or not this was a good idea but it is probably too late to back down now. They come to address you, their face hidden by the now almost total obscurity. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious mood. "We are sorry for all of these complications, but the truth is we cannot afford to take risks. It has been some time since we stopped trying to understand, and started taking every precaution we could to be safe. You see, we simply do not know you, and neither do you us."`
```